screens/Challenges/IndividualChallenge.js

```diff
@@ -0,0 +1,193 @@
+/* this screen shows individual challenges and their submissions */
+
+import React, { useState } from "react";
+import {
+  Image,
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  ScrollView
+} from "react-native";
+import KitText from "../../components/KitText";
+import Colors from "../../constants/Colors";
+import KitBackgroundScreen from "../../components/KitBackgroundScreen";
+export default function IndividualChallenge(props) {
+  const [rating, setRating] = useState("");
+
+  return (
+    <KitBackgroundScreen
+      color={Colors.KIT_RED}
+      onPressBack={() => {
+        props.navigation.navigate("Teams");
+      }}
+      paddinglol={1}
+      title={props.username}
+      number={"1/5"}
+    >
+      <ScrollView>
+        <Image
```

Comment:
Would prefer some light margins around this image to prevent edges from being flush with edges of `KitBackgroundScreen`

Something like this:
![image](https://user-images.githubusercontent.com/23728476/70110981-b1b07300-1605-11ea-8b03-bdde40cbd9fc.png)

In `imageContainer` you could add the line `margin: 15,`
Not blocking

Reply:
ok i think if I remove the padding thing I could get that to work

Reply:
nvm removed the padding option prop and it looked too off so i'm gonna go with ur suggestion